webpack.config.js

```diff
@@ -90,7 +90,8 @@ module.exports = function makeConfig(grunt) {
                 {
                     test: /\.jsx?$/,
                     loader: 'eslint-loader',
-                    exclude: shouldExclude
+                    // superdesk apps handle their own linter
+                    exclude: (p) => shouldExclude(p) || sdConfig.apps.some(app => p.indexOf(app) > -1),
```

Comment:
I apologise. I made a mistake in my comment, I mean to say:
```js
exclude: (p) => p.indexOf('node_modules') !== -1,
```
Really sorry about that :-)